tracer/samples/IISInDocker/src/App_Start/BundleConfig.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Web;
+using System.Web.Optimization;
+
+namespace IISInDocker
+{
+    public class BundleConfig
+    {
+        // For more information on bundling, visit https://go.microsoft.com/fwlink/?LinkId=301862
+        public static void RegisterBundles(BundleCollection bundles)
+        {
+            bundles.Add(new ScriptBundle("~/bundles/jquery").Include(
+                        "~/Scripts/jquery-{version}.js"));
+
+            bundles.Add(new ScriptBundle("~/bundles/jqueryval").Include(
+                        "~/Scripts/jquery.validate*"));
```

Comment:
nit: I would prefer to simplify the sample so we're not committing 50 css, js, and font files that increase repo size without much benefit 🙂

Reply:
Good call! Done in 15f48ad685bcfc1cbb8ca436bf579191419d9dc3